### ISSUE TITLE
Timer for drafting questions

### ISSUE THREAD

**Issue by dmelcer9** [sandboxnu/office-hours#385]
Occasionally, students will leave the drafting window open and step away from the computer. The application should have a (generous) timer that will automatically remove drafting questions from the queue after some time, maybe 10-15 minutes.

**Comment by NEUDitao** [comment]
i agree that this should be done somehow. it's still annoying for TAs to have to close questions of AFK students.
